aopy/preproc/neuropixel.py

```diff
@@ -275,4 +278,86 @@ def destripe_lfp_batch(lfp_data, save_path, sample_rate, bit_volts, max_memory_g
     for ibatch in range(Nbatches):
         tmp = destripe_lfp((lfp_data[ibatch*batch_size:(ibatch+1)*batch_size, :]*bit_volts).T/1e6, sample_rate)*1e6
         lfp_destriped[ibatch*batch_size:(ibatch+1)*batch_size, :] = (tmp/bit_volts).T.astype(dtype)
-        lfp_destriped.flush()
\ No newline at end of file
+        lfp_destriped.flush()
+
+
+def make_average_neuropixel_array(te_id, date, subject, aligning_index, tbefore, tafter):
+    """

```

Comment:
Thanks for including this function but Tomo and I are in the middle of restructuring how LFP preprocessing is done. This will change how data is loaded and stuff so it isn't necessary to include this function in github quite yet.